MoltenVK/MoltenVK/Utility/MVKFoundation.h

```diff
@@ -73,6 +73,8 @@ typedef enum : uint8_t {
 	kMVKCommandUseInvalidateMappedMemoryRanges, /**< vkInvalidateMappedMemoryRanges. */
     kMVKCommandUseBeginRenderPass,              /**< vkCmdBeginRenderPass. */
     kMVKCommandUseNextSubpass,                  /**< vkCmdNextSubpass. */
+	kMVKCommandUseBeginMultiViewPass,           /**< vkCmdBeginRenderPass. */
```

Comment:
This does not appear to be used yet.

Reply:
Done. Good catch. Originally intended to differentiate multi-view from next subpass, but reverted.